src/harness/sas_interface.py

```diff
@@ -256,6 +256,31 @@ def PreloadRegistrationData(self, request):
     """
     pass
 
+  @abc.abstractmethod
+  def InjectExZone(self, request):
+    """Inject PPA or NTIA zone information into SAS under test.
+
+    Args:
+      request: A dictionary with a single key-value pair where the key is
```

Comment:
This does not match the testing API doc.

Reply:
Changed the description